src/amplitude-client.js

```diff
@@ -974,6 +974,14 @@ AmplitudeClient.prototype.setOptOut = function setOptOut(enable) {
   }
 };
 
+/**
+ * Set a custom Session ID for the current session.
+ * Note: This is not recommended unless you know what you are doing because the Session ID of a session is utilized for all session metrics in Amplitude.
+ * The Session ID to set for the current session must be in milliseconds since epoch (Unix Timestamp).
```

Comment:
this is me just not knowing the exact spec here. Is this req'd or just a suggestion?

Reply:
This is actually what was previously documented in our Help Center. If this is not req'd, then we can definitely edit the text.

Reply:
Don't know either. I'll ask the dataPL/Query teams